code/__HELPERS/unsorted.dm

```diff
@@ -875,10 +875,11 @@ var/list/WALLITEMS_INVERSE = typecacheof(list(
 
 		var/list/cached_gases = air_contents.gases
 
-		for(var/id in cached_gases)
-			var/gas_concentration = cached_gases[id][MOLES]/total_moles
-			if(id in hardcoded_gases || gas_concentration > 0.001) //ensures the four primary gases are always shown.
-				user << "<span class='notice'>[cached_gases[id][GAS_META][META_GAS_NAME]]: [round(gas_concentration*100, 0.01)] %</span>"
+		for(var/id = 1 to GAS_LAST)
```

Comment:
@optimumtact it's faster than ```for(var/i = X i < Y; X++)```, both the above are, but they're equivalent to each other (near enough)

so there *are* 3 formats, 2 are fast and one's dogshit, we don't use the dogshit one.